classes/simplemap.php

```diff
@@ -2620,7 +2620,7 @@ function backwards_compat_days_text( $text ) {
 		 * @return string|void
 		 */
 		function backwards_compat_times_text( $text ) {
-			return __( 'Times', 'simplemap' );
+			return __( 'Times', 'SimpleMap' );
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->